File: Programming Basics/JavaScript/For loops/Exercises/tennis-ranklist.js
function solve(input) {
    const n = Number(input[0]);
    const initialPoints = Number(input[1]);

    let newSeasonPoints = 0;
    let wonTournamentsCount = 0;
    for (let i = 0; i < n; i++) {
        const position = input[2 + i];

        if (position == "W") {
            newSeasonPoints += 2000;
            wonTournamentsCount++;
        } else if (position == "F") {
            newSeasonPoints += 1200;
        } else if (position == "SF") {
            newSeasonPoints += 720;
        }
    }

    console.log(`Final points: ${initialPoints + newSeasonPoints}`);
    console.log(`Average points: ${Math.floor(newSeasonPoints / n)}`);
    console.log(`${((wonTournamentsCount * 100) / n).toFixed(2)}%`);
}

File: Programming Basics/C++/For loops/Exercises/TennisRanklist/TennisRanklist.cpp
#include <iostream>
using namespace std;

int main()
{
    int n, initialPoints;
    cin >> n >> initialPoints;

    int newSeasonPoints = 0, wonTournamentsCount = 0;
    for (int i = 0; i < n; i++)
    {
        string position;
        cin >> position;

        if (position == "W")
        {
            newSeasonPoints += 2000;
            wonTournamentsCount++;
        }
        else if (position == "F") { newSeasonPoints += 1200; }
        else if (position == "SF") { newSeasonPoints += 720; }
    }

    cout.setf(ios::fixed);
    cout.precision(2);

    cout << "Final points: " << initialPoints + newSeasonPoints << endl;
    cout << "Average points: " << newSeasonPoints / n << endl;
    cout << (wonTournamentsCount * 100.0) / n << "%" << endl;
}

